iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ThreeWindingsTransformerAdderImpl.java

```diff
@@ -131,7 +131,12 @@ protected TerminalExt checkAndGetTerminal() {
 
         protected VoltageLevelExt checkAndGetVoltageLevel() {
             if (voltageLevelId == null) {
-                throw new ValidationException(this, "voltage level is not set");
+                String defaultVoltageLevelId = checkAndGetDefaultVoltageLevelId();
+                if (defaultVoltageLevelId == null) {
+                    throw new ValidationException(this, "voltage level is not set");
```

Comment:
maybe it could be more explicit, something like "voltage level is not set and it has no default value"